tests/log_parser/test_utils.py

```diff
@@ -0,0 +1,117 @@
+import os
+from copy import deepcopy
+
+import pytest
+import simplejson as json
+from django.db.utils import DataError
+from jsonschema import ValidationError
+
+from treeherder.log_parser.utils import (_lookup_extra_options_max,
+                                         validate_perf_data)
+
+
+@pytest.fixture
+def length_ok():
+    return {
+        'framework': {},
+        'suites': [
+            {
+                'extraOptions': [
+                    '.' * 45,
+                    '.' * 100,
+                ],
+                'name': 'testing',
+                'subtests': []
+            }
+        ] * 3
+    }
+
+
+@pytest.fixture
+def longer_than_all_max():
+    return {
+        'framework': {},
+        'suites': [
+            {
+                'extraOptions': [
+                    '.' * 46,
+                    '.' * 101,
+                ],
+                'name': 'testing',
+                'subtests': []
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def longer_than_max():
+    return {
+        'framework': {},
+        'suites': [
+            {
+                'extraOptions': [
+                    '.' * 45,
+                    '.' * 101,
+                ],
+                'name': 'testing',
+                'subtests': []
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def longer_than_second_max():
```

Comment:
`second_max` is ambiguous. How about `smaller_max`?

Reply:
OK

Reply:
OK